catalog-rest-service/src/main/java/org/openmetadata/catalog/jdbi3/EntityDAO.java

```diff
@@ -49,9 +50,19 @@
   @SqlQuery("SELECT json FROM <table> WHERE id = :id AND deleted IS NOT TRUE")
   String findById(@Define("table") String table, @Bind("id") String id);
 
```

Comment:
We can delete the above variant of methods and only use the new methods you have added

Reply:
That's the plan.

Reply:
Done.